masonite/session.py

```diff
@@ -0,0 +1,47 @@
+
+
+class Session():
+    """
+    Class Session for manage sessions of the app
+    """
+    _session = {}
+
+    def __init__(self, environ):
+        """
+        Constructor
+        """
+        self.environ = environ
+
+    def get(self, key):
+        """
+        Get a session from object _session
+        """
+        ip = self.__get_client_address()
+        try:
+            session = self._session[ip][key]
+        except KeyError:
+            session = ''
+
+        return session
+
+    def set(self, key, value):
+        """
+        Set a new session in object _session
+        """
+        ip = self.__get_client_address()
+        self._session[ip] = {key: value}
```

Comment:
so if you see here, we can only set 1 value to the session. every time you set something it can only contain 1 dictionary in the IP address. 

This test should pass:

```python
def test_session_request():
    REQUEST.session.set('username', 'pep')
    REQUEST.session.set('password', 'secret')
    assert REQUEST.session.get('username') == 'pep'
    assert REQUEST.session.get('password') == 'secret'
```

but it fails because it overwrites the previous set.

This piece of code might have to look something like:

```python
self._session[ip]['key'] = value
```

Now every time you set a different key, it will just keep adding key and value pairs to the session or it will overwrite it.

if that doesn't work you might have to just update the dictionary instead so something like:

```python
self._session[ip].update({key: value})
```

Something like that. Just play with that. Either way that test above should pass. I should be able to set multiple keys on the session